src/Parser/Priest/Holy/Modules/Items/Tier21_2set.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import Analyzer from 'Parser/Core/Analyzer';
+
+const HOLY_PRIEST_TIER21_2SET_BUFF_EXPIRATION_BUFFER = 150; // the buff expiration can occur several MS before the heal event is logged, this is the buffer time that an IoL charge may have dropped during which it will still be considered active.
+
+const HEALING_BONUS = 0.6;
+
+class Tier21_2set extends Analyzer {
+
+  healing = 0;
+  procFH = 0;
+  procGH = 0;
+  procBH = 0;
+
+  on_initialized() {
+      this.active = this.owner.modules.combatants.selected.hasBuff(SPELLS.HOLY_PRIEST_T21_2SET_BONUS_BUFF.id);
+  }
+
+  on_byPlayer_heal(event) {
+      const spellId = event.ability.guid;
+      const hasBuff = this.owner.modules.combatants.selected.hasBuff(SPELLS.HOLY_PRIEST_T21_2SET_BUFF.id, event.timestamp, HOLY_PRIEST_TIER21_2SET_BUFF_EXPIRATION_BUFFER);
+      const isBuffedSpell = ((spellId === SPELLS.GREATER_HEAL.id) || (spellId === SPELLS.FLASH_HEAL.id) || (spellId === SPELLS.BINDING_HEAL_TALENT.id));
+      if (isBuffedSpell && hasBuff) {
+          this.healing += ((event.amount || 0) + (event.absorbed || 0)) / (1 + HEALING_BONUS) * HEALING_BONUS;  
```

Comment:
Please use `calculateEffectiveHealing` (found in src/Parser/Core) for this. It does the same calculation you're trying to do, but also properly handles overhealing.